source/source_labels.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package source
+
+const (
+	sourceControllerName = "sources.knative.dev/controller"
+	sourceName           = "sources.knative.dev/name"
+)
+
+func SourceLabels(name, controllerAgentName string) map[string]string {
```

Comment:
The function name has what Go folks call "stutter" because of the package name: `source.SourceLabels`.  I would just drop the `Source`: `source.Labels`.

This could also use a detailed doc string, since I assume it's being put here for Source authors?